src-plugins/reformat/reformatPlugin.cpp

```diff
@@ -68,60 +42,19 @@ bool reformatPlugin::initialize()
     return true;
 }
 
-bool reformatPlugin::uninitialize()
-{
-    return true;
-}
-
 QString reformatPlugin::name() const
 {
-    return "reformatPlugin";
-}
-
-QString reformatPlugin::description() const
-{
-    return tr("");
+    return "Reformat";
 }
 
 QString reformatPlugin::version() const
 {
     return REFORMATPLUGIN_VERSION;
 }
 
-QString reformatPlugin::contact() const
-{
-    return QString::fromUtf8("Florent Collot <florent.collot@ihu-liryc.fr>");
-}
-
-QStringList reformatPlugin::authors() const
-{
-    return QStringList() << "Mathilde Merle <mathilde.merle@ihu-liryc.fr>"
-                         << "Florent Collot <florent.collot@ihu-liryc.fr>";
-}
-
-QStringList reformatPlugin::contributors() const
-{
-    QStringList list;
-    return list;
-}
-
-QString reformatPlugin::identifier() const
-{
-    return reformatPluginPrivate::s_Name;
-}
-
-
-QStringList reformatPlugin::tags() const
-{
-    return QStringList();
-}
-
 QStringList reformatPlugin::types() const
 {
-    return QStringList() << "reformat";
-}
-QStringList reformatPlugin::dependencies() const
-{
-    return QStringList();
+    return QStringList() << "reformat" << "crop" << "reslice" << "resample";
```

Comment:
Although the ```dtkPlugin::types``` function is not actually used by DTK, I suppose it is meant to return a list of the "types" defined in the plugin. What constitutes a type from DTK's point of view is not explained, but in any case these types are probably designated by their identifiers, so here it should be ```medCropToolBox```, ```resliceToolBox``` etc.